Removed comment containing Non-ASCII character

mmargin opened the pull request:
This change results in ./setup.py test to work properly.

<!-- Your title above should be a short description of what
was changed. Do not include the issue number in the title. -->

#### References to other Issues or PRs
<!-- If this pull request fixes an issue, write "Fixes #NNNN" in that exact
format, e.g. "Fixes #1234". See
https://github.com/blog/1506-closing-issues-via-pull-requests . Please also
write a comment on that issue linking back to this pull request once it is
open. -->


#### Brief description of what is fixed or changed
When running ./setup.py test, the error: 'SyntaxError: Non-ASCII character '\xc2' in file /sympy/sympy/solvers/ode.py on line 1596, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details' appears. Upon locating this error, deleting the comment causes ./setup.py test to run without issue.

#### Other comments


#### Release Notes

<!-- Write the release notes for this release below. See
https://github.com/sympy/sympy/wiki/Writing-Release-Notes for more information
on how to write release notes. The bot will check your release notes
automatically to see if they are formatted correctly. -->

<!-- BEGIN RELEASE NOTES -->
NO ENTRY
<!-- END RELEASE NOTES -->

mmargin committed:
Removed comment containing Non-ASCII character

This change results in ./setup.py test to work properly.

asmeurer commented:
I believe this issue is already fixed in the release branch, which should be merged into master soon CC @oscarbenjamin

mmargin committed:
Delete import on line 15 because not used

When using PyFlakes, this line shows up as having so many unused functions imported. Deleting it will increase overall speed of the program.

mmargin committed:
Import on line 15 has the functions it needs

There were additional imported methods that were unnecessary. I refined them to only include the ones that were needed for the scope of the program.

mmargin committed:
Cleaned up unnecessary imported functions in file

There were additional imported methods that were unnecessary. I refined them to only include the ones that were needed for the scope of the program to run the program faster and increase readability.
